src/models/db.js

```diff
@@ -287,7 +278,13 @@ module.exports = function() {
     foreignKey: 'user_id'
   });
 
-  //belongsToMany - creates an N:M association with a join table and adds plural association mixins to the source. The junction table is created with sourceId and targetId.
+  /**
+   *
+   * @var {attributes} .belongsToMany
```

Comment:
The variable is not called .belongsToMany it is belongsToMany the "." just shows the separation